Shrine.md

```diff
@@ -0,0 +1,6 @@
+The Shrine is an optional activity that all players can access and benefit from weekly. Every week, the shrine will select random buffs that players can offer gold towards to obtain. There can be up to three different buffs that can be selected at random. Each tier of the shrine has an offering goal, that once reached, will be applied at the end of the week. The duration milestones will increase how long the buffs are active, while the buff milestones will give one level of that buff for the duration. This means that if there is five of a single buff within the shrine, you will get five levels of that buff for the duration. (Like Haste 5 for example.) 
+<br />
+In order to make an offering, input the amount of gold you wish to offer in the text box below, then click "OFFER" to confirm. 
+<br />
+<b> Buffs that can appear </b>
+(List of buffs that can appear within the shrine so players can stop asking.)
```

Comment:
Likely to be forgotten aswell, but at least the buffs here don't change a lot

Reply:
I'll remove it then, just some food for thought.